### DISCUSSION TITLE
PCAP file of nFAPI

### DISCUSSION THREAD

**Discussion by anhtrung87vn** [srsran/srsRAN_Project#844] [General Help]
Hi all,

Could you please send me an nFAPI wireshark of an UE attach procedure for reference?

Thanks,
Trung.

**Comment by andrepuschmann** [comment]
Hey - we don't use nFAPI within srsRAN but the non-networked brother FAPI. There is also no PCAP output for it I am afraid.

**Reply by RafalArciszewski** [reply]
We are working on it currently at ITPartners.  Hope we could provide this functionality soon.
@andrepuschmann  can you confirm that srsRAN uses SCF 222.10.04 FAPI version?
